backend/tests/endpoints/test_message.py

```diff
@@ -1,46 +1,75 @@
 from fastapi.testclient import TestClient
 from main import app
+from utils.db import DataStorage
 
 client = TestClient(app)
 
+
 fake_room_id = "44523d5325dsdf"
 fake_org_id = "1223209805sdfsa902"
-fake_message_data = {
-        "sender_id": "425324132312",
-        "room_id": fake_room_id,
-        "text": "does it work well?",
-        "reactions": [],
-        "files": [],
-        "saved_by": [],
-        "is_pinned": False,
-        "is_edited": False,
-        "created_at": "2021-12-11 01:44:37.582879",
-        "threads": []
-        }
+fake_org_id = "42893209343920"
+fake_message_obj = {
+    "sender_id": "425324132312",
+    "room_id": fake_room_id,
+    "org_id": fake_org_id,
+    "message_id": "34o003409",
+    "text": "does it work well?",
+    "reactions": [],
+    "files": [],
+    "saved_by": [],
+    "is_pinned": False,
+    "is_edited": False,
+    "created_at": "2021-12-11 01:44:37.582879",
+    "threads": [],
+}
+DB = DataStorage(fake_org_id)
+
 
-def test_send_message():
-    """ test for the send message endpoint"""
+def test_send_message(monkeypatch):
+    """test for the send message endpoint"""
+
+    def mock_post():
+        return {"status": 201, "message": {"_id": "34928349287395289"}}
+
+    monkeypatch.setattr(DB, "write", mock_post)
 
     fake_response_output = {
-        "message_id": "33090940900",
-        "message_data": fake_message_data
+        "room_id": "61b3fb328f945e698c7eb396",
+        "message_id": "61696f43c4133ddga309dcf6",
+        "text": "str",
+        "files": "HTTP url(s)",
+        "sender_id": "619ba4671a5f54782939d385",
```

Comment:
The sender_id is not the same with the one in the request, this can be another test case when the sender_id is not in the room, the class validator in thread function should have flagged it.